G/Glib/build_tarballs.jl

```diff
@@ -29,13 +30,19 @@ if [[ "${target}" == *-freebsd* ]]; then
 fi
 
 mkdir build_glib && cd build_glib
+
 meson --cross-file="${MESON_TARGET_TOOLCHAIN}" \
     --buildtype=release \
     -Dman=false \
-    -Diconv=external \
     "${MESON_FLAGS[@]}" \
     ..
-ninja -j${nproc}
+
+# Meson beautifully forces thin archives, without checking whether the dynamic linker
+# actually supports them: <https://github.com/mesonbuild/meson/issues/10823>.  Let's remove
+# the (deprecated...) `T` option to `ar`
+sed -i.bak 's/csrDT/csrD/' build.ninja
```

Comment:
Interestingly, this solves also my local problem with `utimensat`.  There's something very weird in this build system...